test/simplifyExpression/arithmeticSearch/arithmeticSearch.test.js

```diff
@@ -12,6 +12,7 @@ describe('evaluate arithmeticSearch', function () {
     ['2*3*5', '30'],
     ['6*6', '36'],
     ['9/4', '9/4'], //  does not divide
+    ['16 + -1953125', '-1953109'], // verify large negative number round correctly
```

Comment:
I think this will work with `16 -1953125` which makes more sense - can you try that?